High Value Tx Update (#345)
NethermindEth/Forta-Agents: anomalous-tx-value/README.md

# Detect anomalous Transaction Value

## Description

> This agent detects transactions with anomalous Transaction Value (i.e. value greater than the `TX_VALUE_THRESHOLD`).

Reviewer: ```suggestion
This bot detects transactions with anomalous Transaction Value (i.e. value greater than a threshold).
```

Removing the `TX_VALUE_THRESHOLD` as it's not defined anymore

> This bot detects transactions with anomalous transaction value (i.e. value greater a threshold).

Reviewer: ```suggestion
This bot detects transactions with anomalous transaction value (i.e. value greater than a threshold).
```

I think this "than" is needed

The thresholds can be set inside `src/agent.config.ts` file.

## Supported Chains

- Ethereum
- Optimism

> - Binance Smart Chain

Author: they are confusing me 😄
